utils/run-topologically/run-topologically.js

```diff
@@ -27,8 +36,11 @@ const TopologicalConfig = figgyPudding({
  * @returns {Promise<Array<*>>} when all executions complete
  */
 function runTopologically(packages, runner, opts) {
-  const { concurrency, graphType, rejectCycles } = TopologicalConfig(opts);
+  const { concurrency, graphType, log, profile, profileLocation, rejectCycles, rootPath } = TopologicalConfig(
```

Comment:
Cool thing about figgy-pudding is that you don't need to make `TopologicalConfig` aware of the profiler options, you can separate those into a `ProfilerConfig` factory that you pass the raw `opts` to instead. (which helps the argument that the profiler itself should be a separate package)

In Profiler.js:
```js
const ProfilerConfig = figgyPudding({
  concurrency: {},
  log: { default: npmlog },
  // alias 'profile' to 'enabled', reads better
  profile: "enabled",
  // a boolean does not need a default, it's implied
  enabled: {},
  profileLocation: {},
  rootPath: {},
};

class Profiler {
  constructor(opts) {
    const { concurrency, log, enabled, profileLocation, rootPath } = ProfilerConfig(opts);
    // ...
```

in run-topologically:
```js
  const profiler = new Profiler(opts);
```

Reply:
Awesome! Thanks!